projects/packages/forms/src/dashboard/inbox/dataviews/index.js

```diff
@@ -401,15 +401,33 @@ export default function InboxView() {
 					onChangeSelection={ onChangeSelection }
 					getItemId={ getItemId }
 					defaultLayouts={ defaultLayouts }
-					header={ <InboxStatusToggle onChange={ resetPage } /> }
 					empty={
 						<EmptyResponses
 							status={ statusFilter }
 							isSearch={ !! view.search }
 							readStatusFilter={ readStatusFilter }
 						/>
 					}
-				/>
+				>
+					<HStack
+						className="jp-forms__inbox__view-actions"
+						spacing={ 2 }
+						alignment="center"
+						justify="space-between"
+					>
+						<HStack spacing={ 2 }>
+							<InboxStatusToggle onChange={ resetPage } />
+						</HStack>
+						<HStack spacing={ 2 } alignment="right">
+							<DataViews.Search />
+							<DataViews.FiltersToggle />
+							<DataViews.ViewConfig />
+						</HStack>
+					</HStack>
+					<DataViews.FiltersToggled className="jp-forms__inbox__filters-container" />
+					<DataViews.Layout />
+					<DataViews.Footer />
+				</DataViews>
```

Comment:
Interesting to me that we can go from 

`<DataViews ... /> `

to 

```
<DataViews> 
  { bunch of stuff in chunks using DataViews.Property }
</DataViews> 
```

like this. I really need to learn more about that package.

Reply:
Interesting and super useful, I didn't know we could do that!

Reply:
😃